schemas/constructs/core.json

```diff
@@ -36,6 +31,406 @@
       "x-go-type-import": {
         "path": "github.com/gofrs/uuid"
       }
+    },
+
+    "styles": {
+      "type": "object",
+      "description": "Common styles for all entities",
+      "additionalProperties": false,
+      "required": ["primaryColor", "svgColor", "svgWhite"],
+      "properties": {
+        "primaryColor": {
+          "type": "string",
+          "description": "Primary color of the component used for UI representation."
+        },
+        "secondaryColor": {
+          "type": "string",
+          "description": "Secondary color of the entity used for UI representation."
+        },
+        "svgWhite": {
+          "type": "string",
+          "description": "White SVG of the entity used for UI representation on dark background."
+        },
+        "svgColor": {
+          "type": "string",
+          "description": "Colored SVG of the entity used for UI representation on light background."
+        },
+        "svgComplete": {
+          "type": "string",
+          "description": "Complete SVG of the entity used for UI representation, often inclusive of background."
+        },
+        "color": {
+          "type": "string",
+          "description": "The color of the element's label. Colours may be specified by name (e.g. red), hex (e.g. #ff0000 or #f00), RGB (e.g. rgb(255, 0, 0)), or HSL (e.g. hsl(0, 100%, 50%))."
+        },
+        "text-opacity": {
+          "type": "number",
+          "description": "The opacity of the label text, including its outline.",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "font-family": {
+          "type": "string",
+          "description": "A comma-separated list of font names to use on the label text."
+        },
+        "font-size": {
```

Comment:
Intentionally, these are not camelCase, right? So long as this is intentional for UI client library compatibility, we're 👍

Reply:
yeah , for cytoscape compatibility .